src/index.js

```diff
@@ -1,43 +1,110 @@
 import './style.css';
 
-const listContainer = document.querySelector('#todo-list');
-
-const LIST = [
-  {
-    description: 'Read DSA',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Finish my To do App',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Learn new JS FrameWorks',
-    completed: false,
-    index: 2,
-  },
-
-];
-const loadTask = () => {
-  const sortedList = LIST.sort((item1, item2) => item1.index - item2.index);
-
-  for (let i = 0; i < sortedList.length; i += 1) {
-    listContainer.innerHTML += `
-    
-    <div class="task-container">
-      <form class="task-form">
-      
-        <input type="checkbox" class="checkbox" <p class="task-description">&nbsp&nbsp${sortedList[i].description} 
-        <i class="fas fa-ellipsis-v pointer edit-task" aria-hidden="true"></i>
-        </p>
-        
-      </form>
-  </div>
-  
-    `;
+let todoTask = [];
+const taskWrapper = document.querySelector('.todo-activities');
+const newTask = document.getElementById('task-description');
+const newTaskBtn = document.getElementById('new-task');
+const reset = document.getElementById('refresh');
+
+const clearallTask = () => {
+  todoTask = [];
+};
+
+const addListToLocalStorage = () => {
+  localStorage.setItem('myTodoTasks', JSON.stringify(todoTask));
+};
+const getListFromLocalStorage = () => {
+  if (localStorage.getItem('myTodoTasks')) {
+    todoTask = JSON.parse(localStorage.getItem('myTodoTasks'));
+  }
+  return todoTask;
+};
+
+const refreshItems = (todoTask) => {
+  for (let i = 0; i < todoTask.length; i += 1) {
+    const indexes = i + 1;
+    todoTask[i].index = indexes;
+  }
+};
+// remove/Delete task
+const removeTask = (index) => {
+  const myLocalStorage = getListFromLocalStorage();
+  myLocalStorage.splice(index - 1, 1);
+};
+// Edit Task
+const editTask = (taskdescription, index) => {
+  for (let j = 0; j < todoTask.length; j += 1) {
+    if (todoTask[j] === index) {
+      todoTask[j] += '*';
+    }
```

Comment:
- [ ] when editing a task you should also save that to localStorage at the moment if I edit a task and reload the page the changes will be gone so please make sure to fix that

Reply:
@youmari  thank you.

- It's now working:

- Before Editing it looks like this

![Screenshot from 2023-01-25 19-04-21](https://user-images.githubusercontent.com/55842414/214614092-61be5521-d6ba-47b8-b9e8-a149018ad40a.png)

- After Editing the task and reloading the page the edited task is saved in LocalStorage

![Screenshot from 2023-01-25 19-08-57](https://user-images.githubusercontent.com/55842414/214614944-9ac280cd-0f93-4f3c-a6a5-85dbd30928d2.png)